### PR TITLE
[CoreLocation] Update to Xcode 9 Beta 1

### PR DESCRIPTION
**Pull request by dalexsoto**

### CODE REVIEW

File: src/corelocation.cs [hunk 1]
```diff
@@ -696,10 +716,31 @@ interface CLGeocoder {
...
 		void CancelGeocode ();
 	}
 
+#if XAMCORE_2_0 && !TVOS
+	[Category]
+	[BaseType (typeof (CLGeocoder))]
+	interface CLGeocoder_ContactsAdditions {
+		[Watch (4,0), NoTV, Mac (10,13), iOS (11,0)]
+		[Export ("geocodePostalAddress:completionHandler:")]
+		// [Async] Generates invalid code: https://bugzilla.xamarin.com/show_bug.cgi?id=57531
```
Comment (spouliot) [inline src/corelocation.cs]: Maybe it's best to have them inlined?

**Comment by dalexsoto** [comment]
@spouliot God idea, fixed 👍

**Comment by dalexsoto** [comment]
mmm keeping this one for reference https://jenkins.mono-project.com/job/xamarin-macios-pr-builder/3995/

**Comment by dalexsoto** [comment]
build

**Pull request merged**